### DISCUSSION TITLE
Format field with other field value

### DISCUSSION THREAD

**Discussion by ewald1976** [filamentphp/filament#1050] [Help] [answered]
Hello,

I have a field "currency" in my table and also a field with a price. I format the "price" field with money.

`Tables\Columns\TextColumn::make('price')->money('EUR'),`

I would like to select the currency dynamically using the "currency" field in my table. Is there a way to access the value from "currency" to format the field?

Thanks in advance.

**Comment by ewald1976** [comment, marked as answer]
Hello,

solved it with
`Tables\Columns\TextColumn::make('price')->getStateUsing(fn ($record): string => money($record->price,$record->currency)),`

**Reply by ArtMin96** [reply]
I needed exactly the same for the same reason :smile:
